man/systemd.mount.xml

```diff
@@ -241,12 +243,10 @@
         <term><option>x-systemd.wanted-by=</option></term>
         <term><option>x-systemd.required-by=</option></term>
 
-        <listitem><para>In the created mount unit, configures a
-        <varname>WantedBy=</varname> or <varname>RequiredBy=</varname>
-        dependency on another unit.  This option may be
-        specified more than once. If this is specified, the normal
-        automatic dependencies on the created mount unit, e.g.,
-        <filename>local-fs.target</filename>, are not automatically
+        <listitem><para>In the created mount unit, configures a <varname>WantedBy=</varname> or
+        <varname>RequiredBy=</varname> dependency on another unit. This option may be specified more than once.
+        If this is specified, the default dependencies (see above) other than <filename>umount.target</filename>
+        on the created mount unit, e.g. <filename>local-fs.target</filename>, are not automatically
```

Comment:
I think it is better to mention that it may be better to also specify x-systemd.after or x-systemd.before, as the default ordering dependencies will be dropped if x-systemd.{wanted,required}-by is specified.